airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml

```diff
@@ -18,93 +18,49 @@ connectionSpecification:
-              const: OAuth Credentials
-              order: 0
-            client_id:
-              title: Client ID
-              description: >-
-                The Client ID of your HubSpot developer application. See the <a
-                href="https://legacydocs.hubspot.com/docs/methods/oauth2/oauth2-quickstart">Hubspot docs</a>
-                if you need help finding this ID.
-              type: string
-              examples:
-                - "123456789000"
-            client_secret:
-              title: Client Secret
-              description: >-
-                The client secret for your HubSpot developer application. See the <a
-                href="https://legacydocs.hubspot.com/docs/methods/oauth2/oauth2-quickstart">Hubspot docs</a>
-                if you need help finding this secret.
-              type: string
-              examples:
-                - secret
-              airbyte_secret: true
-            refresh_token:
-              title: Refresh Token
-              description: >-
-                Refresh token to renew an expired access token. See the <a
-                href="https://legacydocs.hubspot.com/docs/methods/oauth2/oauth2-quickstart">Hubspot docs</a>
-                if you need help finding this token.
-              type: string
-              examples:
-                - refresh_token
-              airbyte_secret: true
-        - type: object
-          title: API key
-          required:
-            - api_key
-            - credentials_title
-          properties:
-            credentials_title:
-              type: string
-              title: Credentials
-              description: Name of the credentials set
-              const: API Key Credentials
-              order: 0
-            api_key:
-              title: API key
-              description: >-
-                HubSpot API Key. See the <a
-                href="https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key">Hubspot docs</a>
-                if you need help finding this key.
-              type: string
-              airbyte_secret: true
-        - type: object
-          title: Private APP
-          required:
-            - access_token
-            - credentials_title
-          properties:
-            credentials_title:
-              type: string
-              title: Credentials
-              description: Name of the credentials set
-              const: Private App Credentials
-              order: 0
-            access_token:
-              title: Access token
-              description: >-
-                HubSpot Access token. See the <a
-                href="https://developers.hubspot.com/docs/api/private-apps">Hubspot docs</a>
-                if you need help finding this token.
-              type: string
-              airbyte_secret: true
+          airbyte_secret: true
 authSpecification:
   auth_type: oauth2.0
   oauth2Specification:
```

Comment:
Because we removed the `oneOf` probably we need to remove the "0" from `rootObject` to looks like
```
    rootObject:
      - credentials
    oauthFlowInitParameters:
      - ["client_id"]
      - ["client_secret"]
    oauthFlowOutputParameters:
      - ["refresh_token"]
```

Reply:
Makes sense, Changing it.